pulsar-metadata/src/main/java/org/apache/pulsar/metadata/api/MetadataStoreConfig.java

```diff
@@ -69,4 +69,10 @@
      */
     @Builder.Default
     private final int batchingMaxSizeKb = 128;
+
+    /**
+     * The operation timeout in seconds.
+     */
+    @Builder.Default
+    private final int operationTimeoutSeconds = 30;
```

Comment:
Will `sessionTimeoutMillis` apply ? Why do we need `operationTimeoutSeconds`?

Reply:
The `sessionTimeoutMillis` is used for heartbeat checks between zk client and server, so I don't think we should use this field as config of operation timeout.

Reply:
Do we have any specific case for "the zk doesn't call the callback"?

Reply:
I made an issue for this, you can try to reproduce this by issue: https://github.com/apache/pulsar/issues/13211. This issue cannot find by the Unit test or Integration test.